packages/buckets/src/api/index.ts

```diff
@@ -34,25 +34,28 @@ import {
   PushPathResponse,
   RemovePathRequest,
   RemoveRequest,
-  Root,
+  Root as _Root,
   RootRequest,
   RootResponse,
   SetDefaultArchiveConfigRequest,
   SetPathRequest,
 } from '@textile/buckets-grpc/api/bucketsd/pb/bucketsd_pb'
-import { APIService, APIServicePushPath } from '@textile/buckets-grpc/api/bucketsd/pb/bucketsd_pb_service'
+import {
```

Comment:
Most of these changes are due to @andrewxhill's prior work on making these types nicer for downstream callers. This work has already been approved.